shared/styles/index.desktop.js

```diff
@@ -133,6 +133,14 @@ export const initDesktopStyles = () => {
     }
   }, '')
   style.appendChild(document.createTextNode(css))
+  style.appendChild(
```

Comment:
this is for us doing programmatically created css, if its static like this just make a .css file